koku/koku/sentry.py

```diff
@@ -25,7 +25,15 @@ def traces_sampler(sampling_context):
     return 0.05
 
 
-if ENVIRONMENT.bool("KOKU_API_ENABLE_SENTRY", default=False):
+if ENVIRONMENT.bool("KOKU_ENABLE_SENTRY", default=False):
+    sentry_sdk.init(
+        dsn=ENVIRONMENT("KOKU_SENTRY_DSN"),
+        environment=ENVIRONMENT("KOKU_SENTRY_ENVIRONMENT"),
+        traces_sampler=traces_sampler,
+    )
+    LOG.info("Sentry setup.")
+# TODO: remove the following in favor of the above `if`
```

Comment:
Do you plan to remove this `TODO` in a followup pr?

Reply:
Yes, these cannot be removed until the other envs are switched to use the new variables.